__tests__/app.test.js

```diff
@@ -458,20 +458,15 @@ describe("GET /api/articles (topic query)", () => {
 });
 
 describe("GET /api/articles/:article_id (comment_count)", () => {
-  it("should return article with comment count if query is set to true", () => {
+  it("should return article including comment count", () => {
     return request(app)
-      .get("/api/articles/1?comment_count=true")
+      .get("/api/articles/1")
       .expect(200)
-      .then(({ body: { article } }) => {
+      .then( async ({ body: { article } }) => {
+        const {rows} = await db.query(`select * from comments where article_id = 1`);
         expect(article).toHaveProperty("comment_count");
-      });
-  });
-  it("should return 400 Bad request if query is present but not allowed", () => {
-    return request(app)
-      .get("/api/articles/1?comment_count=roses")
-      .expect(400)
-      .then(({ body: { msg } }) => {
-        expect(msg).toBe("Bad request");
+        expect(typeof article.comment_count).toBe("number");
+        expect(article.comment_count).toBe(rows.length);
```

Comment:
This is a nice assertion, but simply asserting that it is a number would have also been fine